frontend/src/app/main/ui/workspace/colorpicker.cljs

```diff
@@ -2,27 +2,333 @@
 ;; License, v. 2.0. If a copy of the MPL was not distributed with this
 ;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
 ;;
-;; Copyright (c) 2016-2017 Juan de la Cruz <delacruzgarciajuan@gmail.com>
-;; Copyright (c) 2016-2019 Andrey Antukh <niwi@niwi.nz>
+;; This Source Code Form is "Incompatible With Secondary Licenses", as
+;; defined by the Mozilla Public License, v. 2.0.
+;;
+;; Copyright (c) 2020 UXBOX Labs SL
 
 (ns app.main.ui.workspace.colorpicker
   (:require
    [rumext.alpha :as mf]
    [app.main.store :as st]
-   [app.main.ui.colorpicker :as cp]))
+   [cuerdas.core :as str]
+   [app.util.dom :as dom]
+   [app.util.color :as uc]
+   [app.main.ui.icons :as i]
+   [app.common.math :as math]
+   [app.common.uuid :refer [uuid]]
+   [app.main.data.workspace.libraries :as dwl]
+   [app.main.ui.modal :as modal]
+   [okulary.core :as l]
+   [app.main.refs :as refs]))
+
+(def recent-colors
```

Comment:
Seems like this ref is replaced with `refs/workspace-recent-colors`

Reply:
Ups, I forgot to remove that code. Done